modmod/src/book.rs

```diff
@@ -30,6 +30,8 @@ pub struct Book<'track> {
     pub chapters: Vec<Chapter<'track>>,
 }
 
+const IMAGE_PLACEHOLDER: &str = "#[modmod:images]";
```

Comment:
This is nice. Could you make the other placeholders to consts too?

Reply:
Should be done now